src/dashboard/src/templates/ingest/grid.html

```diff
@@ -234,13 +234,16 @@
       <h3>Upload DIP <img id="upload-dip-modal-spinner" src="/media/images/ajax-loader.gif" /></h3>
     </div>
     <div class="modal-body">
-      <p>Upload the generated DIP to AtoM using the permalink of the target description.</p>
+      <p>Upload the generated DIP to DRMC using a TMS object ID or a technology record slug.</p>
```

Comment:
For this dev branch, we do want this commit, but again we need some additional dev work to refactor this for a more generic approach.  Something like providing a way in the admin page to set this message, or a config file, perhaps.